qubes/ext/__init__.py

```diff
@@ -86,6 +88,9 @@ def decorator(func):
         else:
             func.ha_vm = qubes.vm.BaseVM
 
+        if kwargs.get('is_async', False):
+            func.ha_async = True
+
```

Comment:
`kwargs.get()` in that function was a workaround to get keyword-only arguments in python2 (this is one of the oldest parts of core3).

#TODO rewrite as keyword-only argument. Low priority.